components/Tag/constants/src/index.ts

```diff
@@ -17,6 +17,10 @@ export const CssColor = {
     default: 'bg-gray-50 text-gray-700 border-gray-100',
     dark: 'bg-gray-700 text-white border-gray-700',
   },
+  gray900: {
```

Comment:
@emilmilanov I would recommend calling this one `gray-dark` maybe just to align closer to how we call our `Button` components.

Reply:
I agree.